apps/wallet/configs/webpack/webpack.config.common.ts

```diff
@@ -177,6 +178,9 @@ const commonConfig: () => Promise<Configuration> = async () => {
                 // TODO: check if this is worth investigating a fix and maybe do a separate build for UI and bg?
                 // 'typeof window': JSON.stringify(typeof {}),
                 'process.env.NODE_DEBUG': false,
+                'process.env.WALLET_KEYRING_PASSWORD': JSON.stringify(
+                    Buffer.from(randomBytes(64)).toString('hex')
```

Comment:
It seems to work fine as is, create a new pass every time we start/build the app. Probably session storage will be cleared already and I wanted to avoid adding extra stuff that probably would be unnecessary. Will change it if we see it  invalidates the stored vault in session often

Reply:
Oh that's a good point, yeah in that case don't worry about it.